code/UI/General/TabMenus/TabMenus.cs

```diff
@@ -83,6 +83,12 @@ public override void Tick()
 	[Event.BuildInput]
 	private void MenuInput( InputBuilder input )
 	{
+		if ( !input.Down( InputButton.Score ) )
+		{
+			Input.MouseDelta = Vector2.Zero;
+			Mouse.Position = Screen.Size / 2;
+		}
+
```

Comment:
This makes other menus unusable, like the corpse search menu.

I tested input.Pressed, and it works while the corpse search menu is also open. There's probably some sort of transition period when the cursor is supposed to appear, during which S&box doesn't allow the cursor to be moved.

Reply:
Thanks for reviewing. I should have tested all the UI myself first. I'll see what I can do.

Reply:
Okay so, I've just pushed a change that uses Released instead. This worked in my testing before the PR but it has a quirk/inconsistency in that the cursor won't start from the crosshair on the first time opening the menu for the first time of the session. I was trying to avoid having that inconsistency since stuff like that adds up against end-user QOL but we don't have much choice since it seems like we ran into S&box quirk.